src/index.js

```diff
@@ -0,0 +1,116 @@
+// Include packages needed for this application
+const inquirer = require("inquirer");
+const fs = require("fs");
+
+const generateReadme = require("./generateReadme.js");
+
+//declare a list of questions
+const questions = [
+  //Define the questions
+  //project Name
+  {
+    type: "input",
```

Comment:
It would be nice to include validation for these questions so that they cannot be left blank. Do some research on the `validate` key which is part of the inquirer question object.